src/items/table.jl

```diff
@@ -0,0 +1,4 @@
+struct TabularItem{T} <: DataAugmentation.Item
```

Comment:
```suggestion
struct TabularItem{T} <: Item
```

These don't need to be qualified when they're part of the library itself.